assets/js/modules/optimize/datastore/settings.js

```diff
@@ -0,0 +1,158 @@
+/**
+ * modules/optimize data store: settings.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import API from 'googlesitekit-api';
+import Data from 'googlesitekit-data';
+import dataAPI, { TYPE_MODULES } from '../../../components/data';
+import {
+	isValidOptimizeID,
+	isValidAmpExperimentJSON,
+} from '../util';
+import { STORE_NAME } from './constants';
+
+const { createRegistrySelector, createRegistryControl } = Data;
+
+// Actions
+const SUBMIT_CHANGES = 'SUBMIT_CHANGES';
+const START_SUBMIT_CHANGES = 'START_SUBMIT_CHANGES';
+const FINISH_SUBMIT_CHANGES = 'FINISH_SUBMIT_CHANGES';
+
+export const INITIAL_STATE = {
+	isDoingSubmitChanges: false,
+};
+
+export const actions = {
+	*submitChanges() {
```

Comment:
This is a public action, so it should get a doc-block similar like in e.g. `modules/adsense` - actually that was missed in `modules/analytics` too, if you don't mind would be great to add there as well.

Reply:
Doc-blocked added to optimize and analytics modules submit changes